seqr/views/utils/orm_to_json_utils.py

```diff
@@ -189,12 +198,13 @@ def _process_result(result, family):
     if add_individual_guids_field:
         prefetch_related_objects(families, 'individual_set')
 
+    kwargs = {'additional_model_fields': _get_case_review_fields(families[0], families[0].project, user)}
     if project_guid or not skip_nested:
-        kwargs = {'nested_fields': [{'fields': ('project', 'guid'), 'value': project_guid}]}
+        kwargs.update({'nested_fields': [{'fields': ('project', 'guid'), 'value': project_guid}]})
     else:
-        kwargs = {'additional_model_fields': ['project_id']}
+        kwargs['additional_model_fields'] = (kwargs['additional_model_fields'] or []) + ['project_id']
```

Comment:
Using `kwargs['additional_model_fields'].append('project_id')` as the code for the individual looks nicer.